docs/sources/whatsnew/whats-new-in-v9-1.md

```diff
@@ -0,0 +1,188 @@
+
+#### Starred dashboards in the navigation bar
+
+As part of the upcoming improvements to Grafana's navigation, you can now directly access your [starred dashboards]({{< relref "../dashboards/use-dashboards/" }}) from the navigation bar.
+
+{{< figure src="/static/img/docs/dashboards/starred-dashboards-9-1.png" max-width="750px" caption="Accessing your starred dashboards" >}}
+
+### Panels
+
+#### Heatmap improvements
+
+The beta heatmap announced in version 9.0 is now used throughout Grafana. Its performance is improved, and it now supports [exemplars]({{< relref "../basics/exemplars/" }}). To learn more about the heatmap panel, see the [documentation]({{< relref "../visualizations/heatmap/" >}}).
+
+{{< figure src="/static/img/docs/panels/heatmap-panel-9-1.png" max-width="750px" caption="A heatmap panel" >}}
+
+#### (Beta) Trace to Metrics
+
+You can now link metrics queries to your traces. This feature can be accessed by enabling the `traceToMetrics` feature toggle.
+
+In your tracing datasource configuration, select a metrics datasource, add tags, and write your queries. Each query will appear as a link on each span. The configured tag values will be dynamically added to your metrics queries. You can link to any metric you'd like, and metrics for span durations, counts, and errors filtered by service or span are a great starting point. The metrics generator introduced in Tempo 1.4 pairs extremely well with Trace to Metrics. To learn more about the metrics generator, see the [blog post](https://grafana.com/blog/2022/05/02/new-in-grafana-tempo-1.4-introducing-the-metrics-generator/).
+
+{{< figure src="/static/img/docs/panels/trace-to-metrics-9-1.png" max-width="750px" caption="Linking a trace to a metrics query" >}}
+
+#### (Beta) APM Table
+
+You can now get APM data with Grafana. The data is shown in a handy table in the Tempo datasource under the Service Graph tab. This feature can be accessed by enabling the `tempoApmTable` feature toggle. You will need to enable the metrics generator to receive the data for the APM table.
+
+The APM table displays RED metrics. See your top five request, error, and duration span metrics in the table summary view. The span metrics are created using the Tempo [metrics generator](https://grafana.com/blog/2022/05/02/new-in-grafana-tempo-1.4-introducing-the-metrics-generator/).
+
+We also included several handy links, embedded directly into the table. These links direct you to a Prometheus query to further investigate the data. We also provide a link directly to Tempo search from the table, making it easier for you to investigate your APM metrics.
+
+To learn more about the APM table, see the [documentation]({{< relref "../datasources/tempo" >}}).
+
+{{< figure src="/static/img/docs/panels/apm-table-9-1.png" max-width="750px" caption="An APM table in the Explore view" >}}
+
+### Sharing
+
+#### (Alpha) Public Dashboards
```

Comment:
Is the term Publish Dashboards considered a proper noun?  The capitalization seems to be inconsistent in the text below.

Reply:
Probably not.